.github/workflows/java-ci.yml

```diff
@@ -77,7 +77,7 @@ jobs:
     - uses: actions/setup-java@v1
       with:
         java-version: 8
-    - run: ./gradlew build -x test -x javadoc -x integrationTest
+    - run: ./gradlew -DflinVersions=1.12,1.13 -DsparkVersions=2.4,3.0,3.1,3.2 -DhiveVersions=2,3 build -x test -x javadoc -x integrationTest
```

Comment:
Typo in "flinkVersions" ?

Reply:
Yes, I missed it. Thanks for the finding, @KnightChess !